oggm/utils/_downloads.py

```diff
@@ -657,7 +657,7 @@ def file_downloader(www_path, retry_max=5, cache_name=None,
     return local_path
 
 
-def download_with_authentification(wwwfile, key):
+def download_with_authentication(wwwfile, key):
```

Comment:
"f" is missing here ;-)

Reply:
ah no: "fi"

Reply:
well, I decided to go with the correct spelling :-)
https://english.stackexchange.com/questions/7844/is-authentification-a-real-word

Reply:
Ha! Learned something new, but beh, "authentication" is ugly...